tests/installation/boot_encrypt.pm

```diff
@@ -12,9 +12,21 @@ use strict;
 use warnings;
 use base "installbasetest";
 use utils;
+use version_utils qw(is_sle);
+use Utils::Architectures qw(is_aarch64);
+use testapi;
 
 sub run {
-    unlock_if_encrypted(check_typed_password => 1) if need_unlock_after_bootloader;
+    if (is_aarch64 && is_sle('<=15-SP5') && get_var('FLAVOR', 'Online-QR') && get_var('ENCRYPT', '1') && get_var('FULL_LVM_ENCRYPT', '1')) {
```

Comment:
As this module is owned by Yam squad and we don't test in QU (for now), I would strongly suggest here that you create your own test module as was at the beginning, we don't want to maintain this mysterious conditions, it is hard to read and it will only be worse with time, the code will degenerate with other contributors adding something else.
I believe this could have been done inside `unlock_if_encrypted` as I discussed with you offline (needs to tried), that would be better solution, perhaps for a future ticket in your squad. You could just copy/paste to `boot_encrypt_no_video_present` and have different schedules, once that future ticket is solve you can unify it, but it is up to you (in our experience you can only reduce schedule file with multiple refactors).

Reply:
done